transportccextension_test.go

```diff
@@ -0,0 +1,33 @@
+package rtp
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestTransportCCExtension(t *testing.T) {
+	t1 := TransportCCExtension{}
+
+	rawData := []byte{
+		0x31, 0x00, 0x02, 0x00,
+	}
+
+	if err := t1.Unmarshal(rawData); err != nil {
+		t.Fatal("Unmarshal error on extension data")
+	}
+
+	t2 := TransportCCExtension{
+		ID:  3,
+		L:   1,
+		TSN: 2,
+	}
+
+	if t1 != t2 {
+		t.Error("Unmarshal failed")
+	}
+
+	dstData, _ := t2.Marshal()
+	if !bytes.Equal(dstData, rawData) {
+		t.Error("Marshal failed")
```

Comment:
Could you add a case testing an empty packet returns errTooSmall?

Maybe also worth testing a packet that has extra bytes at the end parses correctly.